code-review-trainer-service/Program.cs

```diff
@@ -52,13 +52,312 @@
         return Results.Ok(Enum.GetNames<DifficultyLevel>());
     }
 
+    // Return a random problem for Easy level
+    if (level == DifficultyLevel.Easy)
+    {
+        var randomProblem = EasyCodeReviewProblems.GetRandomProblem();
+        return Results.Ok(new { level = level.ToString(), problem = randomProblem });
+    }
+
     return Results.Ok($"Test level: {level}");
 })
 .WithName("GetTests")
 .RequireAuthorization();
 
 app.Run();
 
+// Static class containing easy-level code review problems
+public static class EasyCodeReviewProblems
```

Comment:
Please move this into its own file:

`CodeReviewProblems\EasyCodeReviewProblems.cs`